packages/xo-server/src/xapi-object-to-xo.js

```diff
@@ -289,6 +289,7 @@ const TRANSFORMS = {
       affinityHost: link(obj, 'affinity'),
       auto_poweron: otherConfig.auto_poweron === 'true',
       bios_strings: obj.bios_strings,
+      blocked_operations: obj.blocked_operations,
```

Comment:
Do you think we should normalize XO properties and try to use camel case instead of snake case?